src/ItemView.tsx

```diff
@@ -362,7 +362,20 @@ class ItemView extends Component<ItemViewProps, ItemViewState> {
                     if (itemA[sorting.property] === itemB[sorting.property]) return 0;
                     value = itemA[sorting.property] > itemB[sorting.property] ? 1 : -1;
                     break;
-
+                case "use":
+                    let itemAuse = '';
+                    if(itemA.spent) 
+                        itemAuse = 'spent';
+                    else if(itemA.consumed) 
+                        itemAuse = 'consumed';
+
+                    let itemBuse = '';
+                    if(itemB.spent) 
+                        itemBuse = 'spent';
+                    else if(itemB.consumed) 
+                        itemBuse = 'consumed';
+
+                    value = itemAuse.localeCompare(itemBuse);
```

Comment:
_I rewrote the statement in a follow up commit to a simpler ternary expression._

For the future: The if/else if was also acceptable, but please **always** add brackets even if they can be omitted. If you omit them it might lead to a bug in the future.
also always terminate every "case" statement with a "break" for the same reason.